sas/static/webpack/sas/viewer-index.js

```diff
@@ -0,0 +1,302 @@
+import { paginated } from "#core:utils/api";
+import {
+  picturesDeletePicture,
+  picturesFetchIdentifications,
+  picturesFetchPictures,
+  picturesIdentifyUsers,
+  picturesModeratePicture,
+  usersidentifiedDeleteRelation,
+} from "#openapi";
+
+/**
+ * @typedef PictureIdentification
+ * @property {number} id The actual id of the identification
+ * @property {UserProfile} user The identified user
+ */
+
+/**
+ * A container for a picture with the users identified on it
+ * able to prefetch its data.
+ */
+class PictureWithIdentifications {
+  identifications = null;
+  imageLoading = false;
+  identificationsLoading = false;
+
+  /**
+   * @param {Picture} picture
+   */
+  constructor(picture) {
+    Object.assign(this, picture);
+  }
+
+  /**
+   * @param {Picture} picture
+   */
+  static fromPicture(picture) {
+    return new PictureWithIdentifications(picture);
+  }
+
+  /**
+   * If not already done, fetch the users identified on this picture and
+   * populate the identifications field
+   * @param {?Object=} options
+   * @return {Promise<void>}
+   */
+  async loadIdentifications(options) {
+    if (this.identificationsLoading) {
+      return; // The users are already being fetched.
+    }
+    if (!!this.identifications && !options?.forceReload) {
+      // The users are already fetched
+      // and the user does not want to force the reload
+      return;
+    }
+    this.identificationsLoading = true;
+    this.identifications = (
+      await picturesFetchIdentifications({
+        // biome-ignore lint/style/useNamingConvention: api is in snake_case
+        path: { picture_id: this.id },
+      })
+    ).data;
+    this.identificationsLoading = false;
+  }
+
+  /**
+   * Preload the photo and the identifications
+   * @return {Promise<void>}
+   */
+  async preload() {
+    const img = new Image();
+    img.src = this.compressed_url;
+    if (!img.complete) {
+      this.imageLoading = true;
+      img.addEventListener("load", () => {
+        this.imageLoading = false;
+      });
+    }
+    await this.loadIdentifications();
+  }
+}
+
+/**
+ * @typedef ViewerConfig
+ * @param {number} userId Id of the user to get the pictures from
+ * @param {number} albumId Id of the album to displlay
+ * @param {number} firstPictureId id of the first picture to load on the page
+ * @param {bool} userIsSasAdmin if the user is sas admin
+ **/
+
+/**
+ * Load user picture page with a nice download bar
+ * @param {ViewerConfig} Configuration
+ **/
+window.loadViewer = (config) => {
```

Comment:
```suggestion
 * @typedef ViewerConfig
 * @property {number} userId Id of the user to get the pictures from
 * @property {number} albumId Id of the album to displlay
 * @property {number} firstPictureId id of the first picture to load on the page
 * @property {bool} userIsSasAdmin if the user is sas admin
 **/

/**
 * Load user picture page with a nice download bar
 * @param {ViewerConfig} config
 **/
window.loadViewer = (config) => {
```

Il y a d'autres endroits où tu as fait la même erreur.